server/utils/util.go

```diff
@@ -0,0 +1,98 @@
+package utils
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+var l = log.New(os.Stderr, "", log.Ldate)
+
+// Logs error to Stderr.
+func LogError(err error) {
+	l.Println(err)
+}
```

Comment:
nit: this func seems redundant

Reply:
Refactored to return error for invalid plan